visualization/FairnessDashboard/src/Localization/en.json

```diff
@@ -137,8 +140,8 @@
         "r2Description": "The fraction of variance in the labels explained by the model.",
         "aucDescription": "The quality of the predictions, viewed as scores, in separating positive examples from negative examples.",
         "balancedRMSEDescription": "Positive and negative examples are reweighted to have equal total weight. Suitable if the underlying data is highly imbalanced.",
-        "balancedAccuracy": "Balanced accuracy",
         "balancedAccuracyDescription": "Positive and negative examples are reweighted to have equal total weight. Suitable if the underlying data is highly imbalanced."
+        "f1ScoreDescription": "F1-Score is the harmonic mean of precision and recall."
```

Comment:
Any idea why the colon is showing up in red in the GitHub diff?

Reply:
I'm so glad you spotted this! The comma from the previous line was missing. Not sure why I don't see errors when building the widget, though....